src/components/select/select.spec.ts

```diff
@@ -476,6 +542,24 @@ describe('MdlSelect', () => {
 @Component({
     selector: 'test-single-component',
     template: `
+        <mdl-select label="{{label}}" floating-label [autoselect]="false" [(ngModel)]="selectedValue">
```

Comment:
yeah, here a little bit confused naming. cause actually there is feature which might be named as autoselect. I mean when you press any key with character, select going to autoselect it. and what people means under autoselection (maybe proper name autocomplete actually), it's typing some word which filter out selection, and not actually makes autoselection.

so according to it I decided to call original (existing) behavior as autoselect, which can be disabled, so we can use other type of autoselection.